vocabulary.md

```diff
@@ -69,5 +69,5 @@
 ## Y
 ## Z
 
-
+These graphics will be used to instill the common OVN vocabulary across all the OVN Committees and Communities.  
```

Comment:
Hi @jcstine and @Jim1151 - Do you really want to add graphics in the vocabulary markdown file? OR are you using this as a demonstration of the PR process?